compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/todo.jsx-outlining-children.js

```diff
@@ -0,0 +1,36 @@
+// @enableJsxOutlining
+function Component({arr}) {
+  const x = useX();
+  return (
+    <>
+      {arr.map((i, id) => {
+        return (
+          <Bar key={id} x={x}>
+            <Baz i={i}>Test</Baz>
```

Comment:
i'm assuming it's a todo because the "Test" text gets dropped?

Reply:
yes exactly! currently, only jsx as children is outlined